docs/rules/no-private-properties.md

```diff
@@ -0,0 +1,40 @@
+# Disallows usages of "non-public" property bindings (no-private-properties)
+
+It can be helpful to enforce naming conventions for `private` (and sometimes `protected`) members of an object. For example, prefixing private properties with a `__` allows them to be easily discerned when being inspected by tools that do not have knowledge of TypeScript (such as most debuggers).
```

Comment:
this should probably be across multiple lines

Reply:
to be honest i would probably reword to put emphasis on what this rule does, too, rather than about why someone might want visibility naming.

> When following a naming convention to signify the visibility of properties, any which are non-public should
> not be accessed by a consumer of the element. Such properties should be made public and documented
> as such. For example, when following the convention of private properties being prefixed
> by `__` (two underscores) and protected properties being prefixed by `_` (one underscore), it
> should generally be required that only properties without a `__` or `_` prefix are accessed
> from outside the element.

Reply:
Will do this when I get home